network-store-server/src/main/java/com/powsybl/network/store/server/NetworkStoreRepository.java

```diff
@@ -3510,6 +3519,12 @@ private Map<OwnerInfo, List<TapChangerStepAttributes>> getTapChangerStepsWithInC
         if (valuesForInClause.isEmpty()) {
             return Collections.emptyMap();
         }
+        Map<OwnerInfo, List<TapChangerStepAttributes>> v214tapChangerSteps = getV214TapChangerStepsWithInClause(this, networkUuid, variantNum, columnNameForWhereClause, valuesForInClause, variantNumOverride);
+        Map<OwnerInfo, List<TapChangerStepAttributes>> newTapChangerSteps = getNewTapChangerStepsWithInClause(connection, networkUuid, variantNum, columnNameForWhereClause, valuesForInClause, variantNumOverride);
+        return mergeTapChangerSteps(newTapChangerSteps, v214tapChangerSteps);
```

Comment:
It would be nice to just have to delete some code when the migration is done.
See what was done in https://github.com/powsybl/powsybl-network-store-server/commit/1279d22870920eba01f401921a1161d0cc255e55
The idea is to prepare all the work in this PR and to just have to clean the code then